cmd/build/helmify/static/README.md

```diff
@@ -134,13 +134,15 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
 | controllerManager.port                        | Webhook-server port for controller manager                                                                                                                                                                                                          | `8443`                                                                    |
 | controllerManager.metricsPort                 | Metrics port for controller manager                                                                                                                                                                                                                 | `8888`                                                                    |
 | controllerManager.priorityClassName           | Priority class name for controller manager                                                                                                                                                                                                          | `system-cluster-critical`                                                 |
+| controllerManager.podSecurityContext           | Security context on pod level manager                                                                                                                                                                                                          | []                                                 |
```

Comment:
The default value should be a map instead of a list right?

```suggestion
| controllerManager.podSecurityContext           | Security context on pod level manager                                                                                                                                                                                                          | {fsGroup: ..., suplementalGroups: ...}                                                 |
```